api/src/main/java/gov/cms/ab2d/api/config/SwaggerConfig.java

```diff
@@ -42,7 +41,6 @@ public Docket api() {
                 .paths(PathSelectors.ant(API_PREFIX + FHIR_PREFIX + "/**"))
                 .build()
                 .directModelSubstitute(Resource.class, String.class)
-                .directModelSubstitute(JsonNode.class, Void.class)
```

Comment:
I originally did this to prevent `JsonNode` from appearing within the "Model" section of the Swagger doc. Let's make sure that behavior has not re-occurred with this change.

Reply:
I did not see it (and I looked for it). The problem is that it was looking for it and that's why it threw an error. I changed the return type. While I'm not technically returning a "JobCompletedResponse", if it's not completed, it's just populated with null which seems like a reasonable response.